haskell/providers.bzl

```
@@ -31,12 +31,12 @@ HaskellLibraryInfo = provider(
     fields = {
         "package_id": "Workspace unique package identifier.",
         "version": "Package version.",
-        "exports": "List of other `HaskellLibraryInfo` that this package reexports",
+        "exports": "List of `package_id`s that this package exports, that is reexports and this package's own `package_id`",
```

small typo:
```suggestion
        "exports": "List of `package_id`s that this package exports, that it reexports and this package's own `package_id`",
```

---

hm, not a typo but clumsy english. i meant to go for [this idiom](https://www.dictionary.com/browse/that%20is). here are some ideas to make it more clear.

```suggestion
        "exports": "List of `package_id`s that this package exports, that is, reexports and this package's own `package_id`",
```

```suggestion
        "exports": "List of `package_id`s that this package exports, that is to say, reexports and this package's own `package_id`",
```

```suggestion
        "exports": "List of `package_id`s that this package exports, i.e. reexports and this package's own `package_id`",
```